src/fidesops/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -0,0 +1,276 @@
+        raise FunctionalityNotConfigured(
+            "Subject identity verification is required, but it is currently disabled! Please update your application configuration to enable subject identity verification."
+        )
+
+    if not data.email:
+        raise HTTPException(HTTP_400_BAD_REQUEST, detail="An email address is required")
+
+    identity = ProvidedIdentity.filter(
+        db=db,
+        conditions=(
+            (ProvidedIdentity.field_name == ProvidedIdentityType.email)
+            & (ProvidedIdentity.hashed_value == ProvidedIdentity.hash_value(data.email))
+        ),
+    ).first()
+
+    if not identity:
+        provided_identity_data = {
+            "privacy_request_id": None,
+            "field_name": "email",
+            "encrypted_value": {"value": "test@email.com"},
+        }
+        identity = ProvidedIdentity.create(db, data=provided_identity_data)
+
+    consent_request_data = {
+        "provided_identity_id": identity.id,
+    }
+    consent_request = ConsentRequest.create(db, data=consent_request_data)
+    verificaiton_code = _send_verification_code_to_user(db, consent_request, data.email)
+    return ConsentRequestResponse(identity=data, verification_code=verificaiton_code)
+
+
+@router.post(
+    CONSENT_REQUEST_VERIFY,
+    status_code=HTTP_200_OK,
+    response_model=ConsentPreferences,
+)
+def consent_request_verify(
+    *,
+    consent_request_id: str,
+    db: Session = Depends(get_db),
+    data: VerificationCode,
+) -> ConsentPreferences:
+    """Verifies the verification code and returns the current consent preferences if successful."""
+    provided_identity = _get_consent_request_and_provided_identity(
+        db=db, consent_request_id=consent_request_id, verification_code=data.code
+    )
+
+    if not provided_identity.encrypted_value:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND, detail="Provided identity missing email"
+        )
+
+    consent = Consent.filter(
+        db=db, conditions=Consent.provided_identity_id == provided_identity.id
+    ).all()
+
+    identity = (
+        Identity(email=provided_identity.encrypted_value["value"])
+        if provided_identity.field_name == ProvidedIdentityType.email
+        else Identity(phone_number=provided_identity.encrypted_value["value"])
+    )
+
+    if not consent:
+        return ConsentPreferences(identity=identity, consent=None)
+
+    return ConsentPreferences(
+        identity=identity,
+        consent=[
+            Consent(
+                data_use=x.data_use,
+                data_use_description=x.data_use_description,
+                opt_in=x.opt_in,
+            )
+            for x in consent
+        ],
+    )
+
+
+@router.patch(
+    CONSENT_REQUEST_PREFERENCES,
+    status_code=HTTP_200_OK,
+    response_model=ConsentPreferences,
+)
+def set_consent_preferences(
+    *,
+    consent_request_id: str,
+    db: Session = Depends(get_db),
+    data: ConsentPreferencesWithVerificationCode,
```

Comment:
Why is identity part of this request body?  It just seems to be returned directly in the response.

Reply:
It was part of the ticket to include it, but I didn't find a use for it.

Reply:
Alright, but I wouldn't pass identities back and forth for no reason, given that we're pretty careful with not returning identities in other endpoints.

Reply:
@TheAndrewJackson if I remove this will it break anything you have already done?

Reply:
Removing it shouldn't break anything. Having the lists of consents is enough.